bots/pr/src/test/java/org/openjdk/skara/bots/pr/CheckTests.java

```diff
@@ -204,12 +205,15 @@ void multipleReviews(TestInfo testInfo) throws IOException {
             // Approve it
             var reviewerPr = reviewer.pullRequest(authorPr.id());
             reviewerPr.addReview(Review.Verdict.APPROVED, "Reviewers");
+            var nonRecognizedReviewerPr = nonRecognizedReviewer.pullRequest(authorPr.id());
+            nonRecognizedReviewerPr.addReview(Review.Verdict.APPROVED, "");
             TestBotRunner.runPeriodicItems(checkBot);
             assertFalse(authorPr.store().body().contains("Re-review required"));
             assertFalse(authorPr.store().body().contains("Review applies to"));
 
             // Check that it has been approved
             assertTrue(authorPr.store().body().contains("Reviewers"));
+            assertTrue(authorPr.store().body().contains("Reviewers without OpenJDK IDs"));
```

Comment:
I like how the new title relates to the old while clearly indicating the difference between the two lists.